pkgs/development/python-modules/flask-limiter/default.nix

```diff
@@ -33,6 +34,14 @@ buildPythonPackage rec {
     hash = "sha256-3GFbLQExd4c3Cyr7UDX/zOAfedOluXMwCbBhOgoKfn0=";
   };
 
+  patches = [
+    # permit use of rich < 15 -- remove when updating past 3.12
+    (fetchpatch {
+      url = "https://github.com/alisaifee/flask-limiter/commit/008a5c89f249e18e5375f16d79efc3ac518e9bcc.patch";
+      hash = "sha256-dvTPVnuPs7xCRfUBBA1bgeWGuevFUZ+Kgl9MBHdgfKU=";
+    })
+  ];
+
```

Comment:
Try `pythonRelaxDeps = [ "rich" ];`

Reply:
yeah, that works (it's what I used in my personal flake), but was thinking that we'd want something to drive removal when upstream supports new versions? patch will fail to apply which will force the next updater to remove it, and we won't carry an unnecessary `pythonRelaxDeps`. fine either way though -- I can just swap it.

Reply:
Keep it this way then!